sample-webtasks/bundled-webtask/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start":"wt create index --bundle --watch"
+    "start":"wt create --bundle --watch"
```

Comment:
Can you add the `.` at the end of the start script to make it explicit that `wt-cli` should use the current directory. I'm surprised that this would actually work without the file.